File: cpp/39e_1807B_GrabTheCandies.cpp
#include<bits/stdc++.h>

int main(){
    int t, n, m_cnt, b_cnt, temp;
    std::vector<std::string> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n;
        m_cnt = 0, b_cnt = 0;
        for(int j = 0; j < n; j++){
            std::cin>>temp;
            if (temp % 2 == 0){
                m_cnt += temp;
            }
            else{
                b_cnt += temp;
            }
        }
        if(m_cnt > b_cnt){
            ans.push_back("YES");
        }
        else{
            ans.push_back("NO");
        }
    }
    for (auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/39e_1807B_GrabTheCandies.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        arr = input().split()
        m_cnt, b_cnt = 0, 0
        for i in arr:
            if int(i) % 2 == 0:
                m_cnt += int(i)
            else:
                b_cnt += int(i)
        if m_cnt > b_cnt:
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()